### ISSUE TITLE
Payment Going straight to Completed

### ISSUE THREAD

**Issue by 4pixels** [VivaPayments/API#1833]
From what I understand it when Viva OK's a payment and marks it as Complete, my status in WooCommerce goes from 'On Hold' to 'Processing' to 'Completed' instantly and shows that status in my WooCommerce Dashboard. The problem is WooCommerce considers Processing as paid and Completed as when the product has been fulfilled/delivered. 

It's the two definitions of Completed that are a problem, Viva's vs WooCommerce. Our Fulfillment company are using MintSoft, so an order when at Processing triggers their API to instigate a delivery which would then send back to WooCommerce showing a complete status with a delivery number.

In short and this would be the same for any WooCommerce site in that it needs to only set the Complete status themselves manually or via an API when goods have been delivered.

Even a delay between Processing and Complete would be great as MintSoft checks every half hour and could pick up Processing before Completed and trigger the delivery. 

<img width="300" alt="Screenshot 2020-08-25 at 17 46 41" src="https://user-images.githubusercontent.com/1363138/91297644-94749380-e796-11ea-85e9-444ceb1307ce.png">

**Comment by 4pixels** [comment]
I've found the solution under your advanced settings. Completed was the default state, i've changed to Processing
<img width="1030" alt="Screenshot 2020-08-26 at 13 45 05" src="https://user-images.githubusercontent.com/1363138/91306184-1c14cf00-e7a4-11ea-8fb8-bce907ca6652.png">

**Comment by matthewrgourd** [comment]
@4pixels great, glad that solved the problem!
